README.md

```diff
@@ -56,48 +56,27 @@ http://ryanstutorials.net/linuxtutorial/
 
  For RabbitMQ based VOLTTRON, some of the RabbitMQ specific software packages have to be installed.
 
-  **On Debian based systems:**
-
-  If you are running an Ubuntu system, you can install the RabbitMQ dependencies by running the rabbit dependencies
-  script, passing in the approriate distribution as a parameter. Please note,
-  distribution parameter will vary according to the distribution used in your system. For example,
-
-    bionic for Ubuntu 18.04
-    xenial for Ubuntu 16.04
-    xenial for Linux Mint 18.04
-    stretch for Debian Stretch
-    jessie for Debian Jessie
+  **On Debian and CentOS 6/7**
+
+  If you are running an Debian or CentOS system, you can install the RabbitMQ dependencies by running the rabbit 
+  dependencies script, passing in the os name and approriate distribution as a parameter. The following are  
+  supported
+
+    debian bionic (for Ubuntu 18.04)
+    denian xenial (for Ubuntu 16.04)
+    debian xenial (for Linux Mint 18.04)
+    debian stretch (for Debian Stretch)
+    debian jessie (for Debian Jessie)
```

Comment:
Jessie is not supported any longer due to remove of depencency from apt repo for jessie

Reply:
Removed Jessie